tests/unit_tests/test_mesh_from_domain.py

```diff
@@ -16,6 +16,22 @@ def test_reg_mesh_from_cell():
     assert np.array_equal(mesh.upper_right, cell.bounding_box[1])
 
 
+def test_cylindrical_mesh_from_cell():
+    """Tests a CylindricalMesh can be made from a Cell and the specified
+    dimensions are propagated through. Cell is not centralized"""
+    cy_surface = openmc. openmc.ZCylinder(r=50)
+    z_surface_1 = openmc. openmc.ZPlane(z0=30)
+    z_surface_2 = openmc. openmc.ZPlane(z0=0)
```

Comment:
Search and replace gone awry?
```suggestion
    cy_surface = openmc.ZCylinder(r=50)
    z_surface_1 = openmc.ZPlane(z0=30)
    z_surface_2 = openmc.ZPlane(z0=0)
```